assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/index.js

```diff
@@ -42,10 +41,9 @@ import UserCountGraph from './UserCountGraph';
 import DimensionTabs from './DimensionTabs';
 import UserDimensionsPieChart from './UserDimensionsPieChart';
 import { isZeroReport } from '../../../util';
-const { Widget } = Widgets.components;
 const { useSelect } = Data;
 
-function DashboardAllTrafficWidget() {
+function DashboardAllTrafficWidget( { Widget } ) {
```

Comment:
Now that this component also receives the widget component props (in both the new and legacy version, let's also address the TODO comment in line 161/162 and rely on the passed `WidgetReportZero` instead of the imported `ReportZero`.